internal/pkg/deploy/cloudformation/cloudformation_integration_test.go

```diff
@@ -16,13 +16,20 @@ import (
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/aws/session"
 	awsCF "github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/aws/copilot-cli/internal/pkg/aws/iam"
 	"github.com/aws/copilot-cli/internal/pkg/aws/identity"
 	"github.com/aws/copilot-cli/internal/pkg/config"
 	"github.com/aws/copilot-cli/internal/pkg/deploy"
 	"github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation"
 	"github.com/stretchr/testify/require"
 )
 
+const (
+	fmtIAMRoleARN         = "arn:aws:sts::%s:role/%s"
```

Comment:
huh does this need to be "iam" instead of "sts"?

Reply:
lmao you're totally right, this was a dumb typo.